### PR TITLE
Updated Grafana Dashboard creation doc to newer file structure

### PR DESCRIPTION
**Pull request by bryanmontalvan**
Resolves [#1529](https://github.com/operate-first/apps/issues/1529)

### CODE REVIEW

**Review by HumairAK — APPROVED** [review]
/lgtm
/approve

File: docs/content/grafana/add_grafana_dashboard.md [hunk 1]
```diff
@@ -23,15 +23,15 @@ In this template, all you need to do is update:
 1. `metadata.name` - the dashboard name
 2. `spec.url` - the URL pointing to the location of your dashboard JSON file
 
-Pick a suitable `metadata.name`, ensure that its unique in the [`grafana/base/dashboards`][1] folder.
+Pick a suitable `metadata.name`, ensure that its unique in the [`grafana/overlays/moc/smaug/dashboards`][1] folder.
 
-Save this file under `grafana/base/dashboards/${DASHBOARD_FOLDER}/grafana-dashboard-example.yaml`.
+Save this file under `grafana/overlays/moc/smaug/dashboards/${DASHBOARD_FOLDER}/grafana-dashboard-example.yaml`.
```
Comment (4n4nd) [inline docs/content/grafana/add_grafana_dashboard.md:28]: We need to add a new kustomization here.
`grafana/overlays/moc/smaug/dashboards/${DASHBOARD_FOLDER}/kustomization.yaml`

File: docs/content/grafana/add_grafana_dashboard.md [hunk 2]
```diff
@@ -23,15 +23,15 @@ In this template, all you need to do is update:
...
 
-Then add it to `grafana/base/dashboards/kustomization.yaml` by running the following:
+Then add it to `grafana/overlays/moc/smaug/dashboards/kustomization.yaml` by running the following:
 
 ```bash
-$ cd grafana/base/dashboards
+$ cd grafana/overlays/moc/smaug/dashboards
 $ kustomize edit add resource grafana-dashboard-example.yaml
```
Comment (4n4nd) [inline docs/content/grafana/add_grafana_dashboard.md:34]: This should be updated to:
`kustomize edit add resource ${DASHBOARD_FOLDER}`

**Review by 4n4nd — CHANGES_REQUESTED** [review]

File: docs/content/grafana/add_grafana_dashboard.md (hunk 1 above)
Comment (bryanmontalvan) [reply docs/content/grafana/add_grafana_dashboard.md:28]: Just to be sure, you mean replace the path in `line 28` to what you posted? Or to add a new line?

**Pull request merged**